election_2020/model_prediction.py

```diff
@@ -0,0 +1,85 @@
+import os
+import pandas as pd
+import json
+
+from app.BigQuery_service import BigQueryService
+from app.model_storage import ModelStorage
+from app import DATA_DIRPATH, ELECT_FIRPATH
+
+bq = BigQueryService()
+
```

Comment:
new preds table, with schema defined up front (let's add this to the MD file "Setup" section with other migration queries):

```sql

DROP TABLE IF EXISTS `tweet-research-shared.election_2020_transition_2021_combined.openai_embeddings_max_50`;
CREATE TABLE IF NOT EXISTS `tweet-research-shared.election_2020_transition_2021_combined.openai_embeddings_max_50` (

    user_id	INT64,

    is_bot_pred BOOL,
    is_bot_proba ARRAY<FLOAT64>

    -- storing the entire list of probabilities, one for each target (two for binary, four for multiclass)
    --- ... and the remaining targets as well
  
)
```